resources/views/themes/default1/layouts/front/master.blade.php

```diff
@@ -530,31 +527,75 @@
           <script src="{{asset('common/js/intlTelInput.js')}}"></script>
 
         <script>
-         
-    
-                         $.ajax({
-                          type: 'GET',
-                           url: "{{route('twitter')}}",
-                           dataType: "html",
-                             success: function (returnHTML) {
-                                   $('#tweets').html(returnHTML);
-                                    
-                                }
-                          });
-                      
 
-        
-                             function removeItem(id) {
-
-                                             $.ajax({
-                                            type: "GET",
-                                         data:"id=" + id,
-                                    url: "{{url('cart/remove/')}}",
-                                            success: function (data) {
-                                                location.reload();
-                                                                     }
-                                                    });
-                                                    }
+    $('#mailchimp-subscription').click(function(){
```

Comment:
I am not seeing csrf token.
As far as I know, you told you have fixed it. Do you want me to test the backend if it is even validating or we are just sending for the sake of showing off?

Reply:
![Screenshot 2020-05-17 at 7 04 14 AM](https://user-images.githubusercontent.com/31030044/82133698-af854e00-980c-11ea-9727-6a4753155351.png)
Please fix this ASAP. Google the significance and how to exploit it

Reply:
Fixed

Reply:
What fix did you make? I am still not seeing that middleware getting utilized